cms/static/karma_cms.conf.js

```diff
@@ -30,6 +30,7 @@ var options = {
         {pattern: 'cms/**/*spec.js'},
         {pattern: 'coffee/spec/**/*spec.js'},
         {pattern: 'js/certificates/spec/**/*spec.js'},
+        {pattern: 'js/programs/test/specs/*spec.js'},
```

Comment:
LMS/CMS conventions seem to dictate that all spec files should live under js/spec. If you stick with that convention, you won't need to touch this file (the last element in this `specFiles` array has you covered).